src/main/java/Database/Storage.java

```diff
@@ -0,0 +1,182 @@
+package Database; /**
+ * The class Database.Storage handles the memory segment of the Bot
+ */
+import Commands.CommandInvoker;
+import Commands.LoadCommand;
+import Duke.UI;
+import Exceptions.DukeException;
+import Resources.DateTimeHandler;
+import Resources.Task;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedReader;
+
```

Comment:
Coding standard: Order the import statements, maybe move the `import java.util.ArrayList` to the bottom